dbdialog/src/main/java/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -1614,6 +1690,11 @@ protected void getControls() {
     iamSessionToken = (XulTextbox) document.getElementById( "iam-session-token" );
     iamProfileName = (XulTextbox) document.getElementById( "iam-profile-name" );
     namedClusterList = (XulMenuList) document.getElementById( "named-cluster-list" );
+    //azure SQL DB
+    azureSqlJdbcAuthMethod = (XulMenuList) document.getElementById( "azuresql-auth-method-list" );
+    azureAlwaysEncryptionEnabled = (XulCheckbox) document.getElementById( "always-encryption-enable" );
+    azureClientSecretId = (XulTextbox) document.getElementById( "client-id-text" );
```

Comment:
specifically call out the elements for azure-sql.
ex: azure-sql-db-client-id

Same comment for secret key